conmon/conmon.c

```diff
@@ -793,6 +803,23 @@ static gboolean attach_cb(int fd, G_GNUC_UNUSED GIOCondition condition, G_GNUC_U
 	return G_SOURCE_CONTINUE;
 }
 
+/*
+ * resize_winsz resizes the pty window size.
+ */
+static void resize_winsz(int height, int width) {
+	struct winsize ws;
+	int ret;
+
+	ret = ioctl(masterfd_stdout, TIOCGWINSZ, &ws);
```

Comment:
This dates back to 1a682575 (#546), but can we drop it?  In all but the first case, the previous window size will be available in the logs from the last time you set it.  I don't think covering that first case is worth an extra syscall on every resize request.

Reply:
I can drop that.